packages/351elec/sources/scripts/batocera-systems

```diff
@@ -382,9 +382,9 @@ systems = {
 
 
     # NEC
-	"pcengine":   { "name": "PC Engine", "biosFiles":  [ { "md5": "ff1a674273fe3540ccef576376407d1d", "file": "bios/syscard3.pce" } ] },
+	"pcengine":   { "name": "PC Engine", "biosFiles":  [ { "md5": "", "file": "bios/syscard3.pce" } ] },
     "pcfx":       { "name": "PC-FX", "biosFiles":      [ { "md5": "08e36edbea28a017f79f8d4f7ff9b6d7", "file": "bios/pcfx.rom"     } ] },
-    "supergrafx": { "name": "Supergrafx", "biosFiles": [ { "md5": "ff1a674273fe3540ccef576376407d1d", "file": "bios/syscard3.pce" },
+    "supergrafx": { "name": "Supergrafx", "biosFiles": [ { "md5": "", "file": "bios/syscard3.pce" },
```

Comment:
Why don't we add known good md5 sums rather than removing the sum completely?  If there's an opportunity to add multiple hashes, that would be best.

Reply:
Oh I had that first, but that will prevent anyone to launch games with the wrong one. I did some tests today and ES won't start some games if the hash is wrong even if RetroArch is still accepting both version.
The script doesn't accept both else I would have added both.

If you want that we only add the correct one then I'll just revert my last commit.

Reply:
Ahh, it looked like an array or a hash so I assumed we could add multiple without looking deeper into the source.  I'm on the fence here as I would prefer to add a correct hash, but not if that would break other users.

Reply:
If we think that's the case, it seems like this will be fine.

Reply:
Ok fine, but I'll consider if I can extend that script so we can add multiple hashes in the future. :-)